docs/Routes.md

```diff
@@ -103,6 +88,42 @@ fastify.get('/', opts, (req, reply) => {
 
 `fastify.all(path, [options], handler)` will add the same handler to all the supported methods.
 
+<a name="url-building"></a>
+### Url building
+Fastify supports both static and dynamic urls.  
+To register a **parametric** path, use the *colon* before the parameter name. For **wildcard** use the *star*.
```

Comment:
=> ```To register a **parametric** path, use the *colon* (`:`) before the parameter name. For **wildcard** use the *star*` (`*`) ```

Reply:
I don't think this is needed, there is an example few line below and everybody know what is a column and what is a star :P

Reply:
"star" => "asterisk"